@here/harp-mapview/lib/Tile.ts

```diff
@@ -966,6 +966,9 @@ export class Tile implements CachedResource {
         }
         this.textElementsChanged = true;
         this.m_pathBlockingElements.splice(0);
+        this.textElementGroups.forEach((element: TextElement) => {
+            element.dispose();
+        });
```

Comment:
Call to TextElement.dispose() to ensure that it releases it's poi buffer (if any).